framework/core/js/src/forum/components/RenameDiscussionModal.tsx

```diff
@@ -70,8 +81,9 @@ export default class RenameDiscussionModal extends Modal {
           this.loading = false;
           m.redraw();
         });
-    } else {
-      this.hide();
     }
+
+    this.hide();
+    return;
```

Comment:
Do we need this?

Reply:
typescript complains about return type difference in the conditional, I made it explicit instead 🙈.